### PR TITLE
Bitcoin unit: change layout styles and add new icons

### PR DESCRIPTION
**Pull request by MiguelMedeiros**
## Description
New layout to Bitcoin unit screen:  `settings > general > bitcoin unit`

## Preview
https://user-images.githubusercontent.com/5798170/176947800-9ad27be1-73a6-4cfb-b4f0-03a1afbe6719.mp4

### CODE REVIEW

**Review by limpbrains — CHANGES_REQUESTED** [review]
`icons/check.svg` icon is not used

I don't like Idea to put new icons in assets/icons as svg images, since we have all other in .ts files. in this case you can't set custom color to this icon. 

I don't like current approach either, because it is hard to find correct icons. But if we want to switch the approach, we better do it all at once.

You can find bitcoin-unit-satoshis.svg as `<LightningIcon />`

File: src/assets/icons/wallet.ts [hunk 1]
```diff
@@ -5,17 +5,17 @@ export const transferIcon = (
 </svg>`;
 
 export const unitBitcoinIcon = (
-	color = '#ED8452',
+	color = '#FF6600',
 ): string => `<svg width="32" height="32" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
-<path d="M0 16C0 7.16344 7.16344 0 16 0V0C24.8366 0 32 7.16344 32 16V16C32 24.8366 24.8366 32 16 32V32C7.16344 32 0 24.8366 0 16V16Z" fill="${color}" fill-opacity="0.08"/>
```
Comment (limpbrains) [inline src/assets/icons/wallet.ts]: `fill="${color}"` allows you to change icon's color and you removed it, why?

File: src/assets/icons/wallet.ts [hunk 2]
```diff
@@ -5,17 +5,17 @@ export const transferIcon = (
...
+<path d="M0 16C0 7.16344 7.16344 0 16 0V0C24.8366 0 32 7.16344 32 16V16C32 24.8366 24.8366 32 16 32V32C7.16344 32 0 24.8366 0 16V16Z" fill="white" fill-opacity="0.08"/>
+<path d="M12 21.5863H13.688V23H14.592V21.5863H15.744V23H16.64V21.5701C18.76 21.4328 20 20.2856 20 18.4437V18.4276C20 16.9735 19.016 15.9233 17.488 15.7536V15.689C18.6 15.4951 19.456 14.4368 19.456 13.2493V13.2331C19.456 11.674 18.36 10.5995 16.64 10.438V9H15.744V10.4137H14.592V9H13.688V10.4137H12V21.5863ZM13.752 15.1962V11.8679H15.736C17.008 11.8679 17.712 12.4414 17.712 13.4674V13.4836C17.712 14.5903 16.896 15.1962 15.376 15.1962H13.752ZM13.752 20.1321V16.5776H15.72C17.408 16.5776 18.192 17.1431 18.192 18.3226V18.3387C18.192 19.5263 17.416 20.1321 15.824 20.1321H13.752Z" fill="${color}"/>
 </svg>`;
 
 export const unitSatoshiIcon = (
-	color = '#ED8452',
+	color = '#FF6600',
 ): string => `<svg width="32" height="32" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
-<path d="M0 16C0 7.16344 7.16344 0 16 0C24.8366 0 32 7.16344 32 16C32 24.8366 24.8366 32 16 32C7.16344 32 0 24.8366 0 16Z" fill="${color}" fill-opacity="0.08"/>
```
Comment (limpbrains) [inline src/assets/icons/wallet.ts]: same here

**Review by limpbrains — CHANGES_REQUESTED** [review]

File: src/assets/icons/wallet.ts (hunk 1 above)
Comment (MiguelMedeiros) [reply src/assets/icons/wallet.ts]: Ops! My bad! Will fix it!

File: src/assets/icons/wallet.ts (hunk 2 above)
Comment (MiguelMedeiros) [reply src/assets/icons/wallet.ts]: Ops! My bad! Will fix it!

**Comment by limpbrains** [comment]
also remove unused svg files

**Review by limpbrains — APPROVED** [review]

**Review by Jasonvdb — APPROVED** [review]

**Pull request merged**